packages/compass-aggregations/test/configure-store.ts

```diff
@@ -35,6 +36,9 @@ export default function configureStore(
       localAppRegistry: new AppRegistry(),
       workspaces: {} as any,
       logger: createNoopLoggerAndTelemetry(),
+      pipelineStorage: {
+        loadAll: () => Promise.resolve([]),
+      } as unknown as PipelineStorage,
```

Comment:
Passing a bare minimum here only because it was the easiest thing to do. Let me know we would rather prefer having a full mock implementation of PipelineStorage.

Reply:
I think I was trying to avoid a @tslint-ignore here since we [don't allow imports](https://github.com/mongodb-js/compass/blob/COMPASS-7411-provider-for-my-queries-storage/configs/eslint-config-compass/plugin.js#L23) from `@mongodb-js/my-queries-storage`.
I understand that this should be fine for test files/test helpers here but as per our [override definition](https://github.com/mongodb-js/compass/blob/COMPASS-7411-provider-for-my-queries-storage/configs/eslint-config-compass/index.js#L40-L58) this is not a test file. How about I expand that definition to turn the rule off also for the entire test directory, including something like `**/test/**/*`?

Reply:
> How about I expand that definition to turn the rule off also for the entire test directory, including something like `**/test/**/*`?

Yes, that makes sense to me and is definitely in line with the intentions of those definitions 👍